sklearn/datasets/tests/test_kddcup99.py

```diff
@@ -44,3 +45,37 @@ def test_shuffle(fetch_kddcup99_fxt):
     dataset = fetch_kddcup99_fxt(random_state=0, subset='SA', shuffle=True,
                                  percent10=True)
     assert(any(dataset.target[-100:] == b'normal.'))
+
+
+def test_asframe(fetch_kddcup99_fxt):
+    pd = pytest.importorskip('pandas')
+
+    data = fetch_kddcup99_fxt(as_frame=True)
+    fetch_func = partial(fetch_kddcup99_fxt, 'smtp')
+
+    assert hasattr(data, 'frame') is True
+    assert check_as_frame(data, fetch_func)
```

Comment:
So you should not need this anymore because we introduced some common tests.

Reply:
OK, will get rid of this.